src/js/form.js

```diff
@@ -251,6 +301,10 @@ Form.prototype.init = function() {
     return loadErrors;
 };
 
+/**
+ * @param {string} xpath
+ * @return {Array<Error>} A list of errors originated from `goToTarget`. Empty if everything went fine.
+ */
```

Comment:
This is an array of error messages (strings) not Error instances

Reply:
Fixed